File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    int i, j, k;

    for (i = 0; i < height; i++)
    {
        for (j = (height - 1); j > i; j--)
        {
            printf(" ");
        }
        for (k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/sentimental-mario-less/mario.py
from cs50 import get_int

height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    count = 0
    blank = height - 1
    while blank > i:
        blank -= 1
        print(" ", end="")
    while count <= i:
        count += 1
        print("#", end="")
    print("")
